app/router.go

```diff
@@ -96,6 +97,9 @@ func makeReportPostHandler(a xfer.Adder) http.HandlerFunc {
 			return
 		}
 		a.Add(rpt)
+		if len(rpt.Pod.Nodes) > 0 {
+			addKubernetesTopologies()
+		}
```

Comment:
Yeah, agree with @tomwilkie here, the dynamism feels a bit awkward... will have to think a bit on an alternative...